### PR TITLE
Improve experience for advanced users (RSSI, altitude, auto_reconnect fixes)

### PR DESCRIPTION
**Pull request by jstockdale**
* Add RSSI to debug messages that have rx_rssi available
* Fix altitude handling for client (and support ft or meters)
* Improve auto_reconnect handling

### CODE REVIEW

**Comment by pdxlocations** [comment]
Sweet!  I'll try to have a look this evening after work.

**Comment by jstockdale** [comment]
Ok this version of auto_reconnect actually works–
image

I'll keep improving it but the first version usually failed to reconnect properly (and gave a channel changed error). Progress!

File: meshtastic-mqtt-connect.py [hunk 1]
```diff
@@ -637,10 +648,15 @@ def send_position(destination_id):
         latitude_i = int(latitude)
         longitude_i = int(longitude)
 
+        altitude_str = alt_entry.get()
+        altitude_units = 3.28084 if 'm' in altitude_str else 1.0
+        altitude_number_of_units = float(re.sub('[^0-9.]','', altitude_str))
+        altitude_i = int(altitude_units * altitude_number_of_units)
```
[the remark this answers is not in the document]
Comment (pdxlocations) [reply meshtastic-mqtt-connect.py:654]: Adding an 'm' also isn't obvious to the user.  I think we'd need a radio button or a check box 'is_feet' or the like.

[the remark this answers is not in the document]
Comment (jstockdale) [reply meshtastic-mqtt-connect.py:654]: My stupid T-Beam gps thinks it’s in geosynchronous orbit half the time 😐

[the remark this answers is not in the document]
Comment (jstockdale) [reply meshtastic-mqtt-connect.py:654]: 😂😂😂

[the remark this answers is not in the document]
Comment (pdxlocations) [reply meshtastic-mqtt-connect.py:654]: devices display in feet/miles if you set your display to IMPERIAL but it's always sent in meters over the mesh.

**Comment by jstockdale** [comment]
Oh interesting. Thanks for that info. I’ll check what’s going on in the
code and fix this PR tonight. :-)

**Comment by jstockdale** [comment]
Ok this should fix the altitude handling in a sensible way.

**Comment by jstockdale** [comment]
Let me know if you'd like any other fixed. I synced this branch with the main repo so it's current again.

**Pull request merged**